addons/api/mirage/factories/managed-group.js

```diff
@@ -0,0 +1,20 @@
+import factory from '../generated/factories/managed-group';
+import permissions from '../helpers/permissions';
+
+export default factory.extend({
+  id: (i) => `managed-groups-id-${i}`,
+
+  authorized_actions: () =>
+    permissions.authorizedActionsFor('managed-group') || [
+      'no-op',
+      'read',
+      'update',
+      'delete',
+    ],
+
+  authorized_collection_actions: () => {
```

Comment:
Managed groups do not have sub collections.

Reply:
@randallmorey 
So `authorized_collection_actions` does not apply here?

But I should apply `create` and `list` permissions, right?

Reply:
To mock permission to `create` and `list` `managed-groups`, add an entry to the `auth-method` factory.  `authorized_collection_actions` defines what permissions the user has on _sub-collections_.  There are no sub-collections under managed groups itself, so it doesn't not need this field.